packages/xo-server-load-balancer/src/index.js

```diff
@@ -14,6 +15,11 @@ const DENSITY_MODE = 1
 
 // ===================================================================
 
+// Maximum number of concurrent migrations
+const MAX_CONCURRENT_MIGRATIONS = 2
```

Comment:
Make it configurable as an advanced setting.

Feel free to use `auth-oicd` implementation as an exemple: https://github.com/vatesfr/xen-orchestra/blob/b7a66e9f73884cfcaf4779a424bb583acd73746e/packages/xo-server-auth-oidc/index.js#L35-L40

Reply:
I did not make it configurable in the first place because I thought it could be confusing, for example if it's configured to 5 concurrent migrations in the plugin, but the XO limit prevents more than 3 concurrent migrations with storage motion. But if you think this case will probably not happen that much, then we can make it configurable

Reply:
It needs to be configurable, even if only in the `staticConfig`.

Sometimes it's necessary to change this value in a specific case (maybe setting it `1`, maybe when doing support for a customer).

Rule of thumb: always avoid magic values in code.